### ISSUE TITLE
Compilation warning to deal with

### ISSUE THREAD

**Issue by russel** [AlbrechtL/welle.io#140]
When compiling with GCC7 on Debian Sid and Fedora Rawhide, there is the warning:
```
/home/users/russel/Repositories/Git/Forks/WelleIo/src/backend/fib-processor.cpp: In constructor ‘fib_processor::fib_processor(CRadioController*)’:
/home/users/russel/Repositories/Git/Forks/WelleIo/src/backend/fib-processor.cpp:105:27: warning: ‘memset’ used with length equal to number of elements without multiplication by element size [-Wmemset-elt-size]
     memset (dateTime, 0, 8);
                           ^
```

**Comment by AlbrechtL** [comment]
Thanks for reporting!

**Comment by AlbrechtL** [comment]
Fixed in fea3a21.
